tests/test_docker_extract.py

```diff
@@ -17,6 +18,15 @@ def test_traverse_workflow(self):
 
         with TemporaryDirectory() as tmpdir:
             for req in set(traverse(loaded)):
-                image_name = get_image_name(req)
-                save_docker_image(req, image_name, tmpdir)
-                _ = load_docker_image(image_name)
+                image_puller = DockerImagePuller(req, tmpdir)
+                image_puller.save_docker_image()
+                _ = image_puller.generate_udocker_loading_command()
+
+    @skipIf(TRAVIS, reason="travis doesn't running docker in singularity")
+    def test_traverse_workflow_singularity(self):
```

Comment:
FYI: https://github.com/common-workflow-language/cwltool/blob/master/.travis.yml#L6

Reply:
Thanks. I am not sure about the tests, but it seems that it was already failing previously on Travis?

Reply:
@mr-c do you think we could merge this PR or there is something to fix? I am not sure I understood your comment since tests were not working before neither. Cheers